src/hazelcore/UHExp.rei

```diff
@@ -17,10 +17,12 @@ and operand =
   | IntLit(ErrStatus.t, string)
   | FloatLit(ErrStatus.t, string)
   | BoolLit(ErrStatus.t, bool)
+  | StringLit(ErrStatus.t, string)
```

Comment:
string argument here should really be of type UnescapedString.t, right?

Reply:
No, since the strings in UHExp are unparsed. They're parsed during elaboration. Maybe "unescaped" isn't the clearest naming, though.

Reply:
Oh I see, the name makes sense but maybe we should also define EscapedString.t so its clear which is which